src/pykiso/lib/connectors/cc_pcan_can.py

```diff
@@ -399,115 +421,80 @@ def _merge_trc(self) -> None:
                 # If a log file is not provided, take the fist trace created as result file
                 result_trace = list_of_traces[0]
             else:
-                # Else write the first trace content in the log file and then remove it
+                # Rename the trace if necessary
                 result_trace = Path(self.trace_path / self.trace_name)
-                list_of_traces[0] = shutil.move(
-                    str(list_of_traces[0]), str(result_trace)
-                )
-                list_of_traces[0] = Path(list_of_traces[0])
-
-            # End of the trace header
-            first_message_line = 33
 
-            # Get start time of the first trc file
-            with list_of_traces[0].open("r") as trc:
-                data = trc.read().splitlines(True)
-
-                first_trc_start_time = CCPCanCan._get_trace_start_time(
-                    data, first_message_line
+                list_of_traces[0] = Path(
```

Comment:
Maybe I'm wrong, but this seems to be unnecessary as the content is overwritten afterwards, to me this does:
- rename the first trc to the expected name
- overwrite the first trc with the first trc's header, content (?) and the content of the other trcs

Reply:
Resolved, this is done to ensure that all of the other traces are deleted